src/tictactoe/game_menu.clj

```diff
@@ -0,0 +1,59 @@
+(ns tictactoe.game-menu
+  (:require [clojure.string :as string]
+            [tictactoe.user-interface :as ui]
+            [tictactoe.game-messages :as messages]
+            [tictactoe.menu-validation :as validator]
+            [tictactoe.marker-validation :as marker-validator]))
+
+(defn welcome-message []
+  (ui/print-message (messages/welcome)))
+
+(defn pick-game [game-types] 
+  (ui/print-message (messages/game-type))
+  (ui/loop-and-print game-types)
+  (-> 
+    (ui/get-input)
+    (keyword)
+    (validator/input-validation-loop game-types)
+    (keyword)))
+
+(defn pick-board-size [board-sizes] 
```

Comment:
`pick-game` and `pick-board-size` functions seem identical except the parameter that's passed in and the message. Can that be abstracted?